client/src/app/components/login-form/login-form.component.html

```diff
@@ -0,0 +1,38 @@
+<form (ngSubmit)="onSubmit()" class="login-form">
+  <ion-item>
+    <ion-label position="stacked">E-mail</ion-label>
+    <ion-input
+      name="email"
+      type="email"
+      required
+      inputmode="email"
+      pattern="[a-zA-Z0-9\.]+@[a-zA-Z0-9]+\.[a-zA-Z0-9]+"
+      autocomplete="email username"
+      clearInput="true"
+      data-test="input-user"
+    ></ion-input>
+  </ion-item>
+  <ion-item>
+    <ion-label position="stacked">Password</ion-label>
+    <ion-input
+      name="password"
+      type="password"
+      required
+      autocomplete="current-password"
+      minlength="4"
+      clearInput="true"
+      clearOnEdit="false"
+      data-test="input-password"
+    ></ion-input>
+  </ion-item>
+  <ion-button
+    buttonType="button"
+    type="submit"
+    class="login-button"
+    expand="block"
+    data-test="login-button"
+  >
+    Log In
+    <ion-icon slot="end" name="enter"></ion-icon>
```

Comment:
```suggestion
    <ion-icon slot="end" name="enter" aria-hidden="true"></ion-icon>
```

If there is already a text-label in the button, the visual icon can be hidden.